hls/lib/hls_endpoint.ex

```diff
@@ -509,6 +560,13 @@ defmodule Membrane.RTC.Engine.Endpoint.HLS do
 
   defp get_hls_stream_directory(state, _stream_id), do: state.output_directory
 
+  defp log_skipped_track(track_id) do
+    Membrane.Logger.debug("""
+    Couldn't subscribe to the track: #{inspect(track_id)}.
+    No such track.
+    """)
+  end
```

Comment:
What do you think about inlining this function?

Reply:
this function is called in two places and also makes it easier to read `handle_info(:subscribe_track...` and `add_track`

Reply:
I think it is strange that we used the same log in two places for the same thing. Maybe `add_tracks` should work on `track_ids` and return the ids on which it successfully subscribes? And then outside of this function, we would put tracks structs into state?

Reply:
`add_tracks` would do unnecessary `Endpoint.subscribe` calls. I will try to fix it somehow.